src/Php72/composer.json

```diff
@@ -1,32 +0,0 @@
-{
-    "name": "symfony/polyfill-php72",
-    "type": "library",
-    "description": "Symfony polyfill backporting some PHP 7.2+ features to lower PHP versions",
-    "keywords": ["polyfill", "shim", "compatibility", "portable"],
-    "homepage": "https://symfony.com",
-    "license": "MIT",
-    "authors": [
-        {
-            "name": "Nicolas Grekas",
-            "email": "p@tchwork.com"
-        },
-        {
-            "name": "Symfony Community",
-            "homepage": "https://symfony.com/contributors"
-        }
-    ],
-    "require": {
-        "php": ">=7.1"
```

Comment:
In #428, I did not remove the package just yet, but instead bumped it to PHP 7.2 as well and made it a no-op package. We've done this with other polyfills in the past as well, e.g. https://packagist.org/packages/symfony/polyfill-php71#v1.20.0

Reply:
why did we do this? what's the benefit?
people with php < 7.2 can still install a previous version